_posts/2018-03-23-cache-breaking-for-faster-web-application-updates.md

```diff
@@ -10,26 +10,28 @@ tags: []
+Recently, the URL changed for one of the map services we use in several of our applications. This came as a bit of a surprise to us, so we quickly updated our applications and deployed new versions. Everything seemed great and we continued on our way. However, two days later one of our clients reported that some of his users still did not have the updated applications in their browsers. Walking clients through how to clear their browser caches felt unprofessional—like I'd be giving them an unnecessary, and possibly unwelcome, explanation of a simple subject. (Obviously not the ideal solution.) So I started doing some digging on how I could solve this problem behind the scenes. 
 
 ### Head[er]aches
 Here's an example of a request for the main JS file for our application:
 
 ![screenshot of headers]({{ '/images/cache_headers.png' | prepend: site.baseurl }}){: .flex .flex--center .outline }
 
-Notice `(from disk cache)` next to the status code? That means that the browser wasn't even making a request to the server for this file. This is really great for speeding up the loading of the application but not so great when you are trying to get updates to users' browsers. The problem with this request is that because it specifies neither the `Cache-Control` nor `Expires` header, the browser is left on its own to guess at how long it should cache the resource. This can vary between browsers and, in general, is a bad idea. But we also don't want to completely disable caching either. It can make a huge difference on page load speed. Google's Web Fundamentals site has [a really great article on this subject](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching). This article not only helped me understand the problem but gave me the direction on a solution.
+Notice the "`(from disk cache)`" next to the status code? That means that the browser wasn't even making a request to the server for this file. This is really great for speeding up the loading of the application, but it's not so great when you're trying to get updates to users' browsers. The problem with this request is that because it doesn't specify the `Cache-Control` or `Expires` header, the browser is left playing a guessing game about how long it should cache the resource. The length of time a browser will cache a resource can vary between browsers, so, in general, going along with this guessing game is a bad idea. But we also don't want to completely disable caching either&mdash;it can make a huge difference on page load speed. (Google's Web Fundamentals site has [a really great article on this subject](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching). This article not only helped me understand the problem, it gave me direction on a solution.)
```

Comment:
How's this rewording about "the browser is left playing a guessing game..."?

Reply:
seems fine